source/nodejs/adaptivecards-designer/src/open-sample-dialog.ts

```diff
@@ -19,7 +19,9 @@ class CatalogueItem {
     }
 
     render(): HTMLElement {
-        const newItemId = CatalogueItem.getNewItemId("acd-open-sample-item-title");
+        const newItemId = CatalogueItem.getNewItemId(
```

Comment:
This really should be on a single line...

Reply:
@dclaux  Addressed the review comments by changing to single line.  This has been changed due to Prettier automatic formatting.